### ISSUE TITLE
loss is 341

### ISSUE THREAD

**Issue by rock3220182** [sribdcn/anoymous_MIP_method#5]
Dear Authors,

I hope this message finds you well. Recently, I've had the privilege to work on the implementation of your project. I have been diligently conducting experiments as per the method outlined in your paper. For my setup, I utilized the IP dataset with 240 instances for training and 60 instances for validation. I've set the batch size at 8, maxStoredSol at 500, and a learning rate of 0.003. I've noticed that the model's loss stabilizes around 341 without further decrease. I was wondering if this might be attributed to my choice of hyperparameters or perhaps the size of the training dataset.

I would greatly appreciate your insights or suggestions on this matter. Thank you for your time and consideration.

Warm regards,

Transtars

**Comment by haowang-tech** [comment]
@rock3220182 hello, I met the same problem when I trained the model with my insatnce data.  I also adjusted the parameters, but the loss did not improve

**Comment by RuohLiuq** [comment]
@rock3220182
Hi, I met the same problem and it's also 341. Have you solved this problem?:)

**Comment by elender9098** [comment]
Hi bro! Can you reproduce the results in the paper?

**Comment by ericli2333** [comment]
Hey bro, can anyone tell me how to handle this? I've tried to use the `GNNPolicy_position` but that doesn't work.

**Comment by elender9098** [comment]
Nah bro, tried that already. That joint still broken, so I’ve moved on to other stuff for now.

**Comment by haowang-tech** [comment]
I think it's quite normal for this situation to occur, because predicting different types of variables leads to very low accuracy and high loss.
